tgui/packages/tgui/interfaces/Pandemic.tsx

```diff
@@ -0,0 +1,484 @@
+        </Stack.Item>
+        <Stack.Item>
+          <SymptomInfoDisplay symptoms={virus.symptoms} />
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+/** Displays info about the virus.
+ * Note: Players are only getting one of these from DM,
+ * but the functionality is here to display more than one.
+ */
+const VirusInfoDisplay = (props: VirusInfoProps, context) => {
+  const { act } = useBackend<PandemicContext>(context);
+  const { virus } = props;
+
+  return (
+    <Stack fill>
+      <Stack.Item grow={3}>
+        <LabeledList>
+          <LabeledList.Item label="Name">
+            {virus.can_rename ? (
+              <Input
+                value={virus.name}
+                onChange={(e, value) =>
+                  act('rename_disease', {
+                    index: virus.index,
+                    name: value,
+                  })}
+              />
+            ) : (
+              <Box color="bad">{virus.name}</Box>
+            )}
+          </LabeledList.Item>
+          <LabeledList.Item label="Description">
+            {virus.description}
+          </LabeledList.Item>
+          <LabeledList.Item label="Agent">
+            {virus.agent.replace(/^\w/, (c) => c.toUpperCase())}
+          </LabeledList.Item>
+          <LabeledList.Item label="Spread">{virus.spread}</LabeledList.Item>
+          <LabeledList.Item label="Possible Cure">
+            {virus.cure}
+          </LabeledList.Item>
+        </LabeledList>
+      </Stack.Item>
+      {virus.is_adv && (
+        <>
+          <Stack.Divider />
+          <Stack.Item grow={1}>
+            <Section title="Statistics">
+              <LabeledList>
+                <Tooltip content="Decides the cure complexity.">
+                  <LabeledList.Item
+                    color={GetColor(virus.resistance)}
+                    label="Resistance">
+                    {virus.resistance}
+                  </LabeledList.Item>
+                </Tooltip>
+                <Tooltip content="Symptomic progression.">
+                  <LabeledList.Item
+                    color={GetColor(virus.stage_speed)}
+                    label="Stage speed">
+                    {virus.stage_speed}
+                  </LabeledList.Item>
+                </Tooltip>
+                <Tooltip content="Detection difficulty from medical equipment.">
+                  <LabeledList.Item
+                    color={GetColor(virus.stealth)}
+                    label="Stealth">
+                    {virus.stealth}
+                  </LabeledList.Item>
+                </Tooltip>
+                <Tooltip content="Decides the spread type.">
+                  <LabeledList.Item
+                    color={GetColor(virus.transmission)}
+                    label="Transmissibility">
+                    {virus.transmission}
+                  </LabeledList.Item>
+                </Tooltip>
+              </LabeledList>
+            </Section>
```

Comment:
Same again. This section can probably be pulled out into its own component for readability.

Reply:
Done